lib/fs.js

```diff
@@ -1112,14 +1114,14 @@ function writeAll(fd, buffer, offset, length, position, callback) {
 }
 
 fs.writeFile = function(path, data, options, callback) {
-  var callback = maybeCallback(arguments[arguments.length - 1]);
+  callback = maybeCallback(arguments[arguments.length - 1]);
```

Comment:
@thefourtheye I'm using this test: https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#1-tooling